src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -286,7 +286,34 @@ private Cipher(CipherSpi firstSpi, Service firstService,
         this.lock = new Object();
     }
 
-    private static final String SHA512TRUNCATED = "SHA512/2";
+    // for special handling SHA-512/224, SHA-512/256, SHA512/224, SHA512/256
+    private static final String SHA512TRUNCATED = "512/2";
```

Comment:
Is this constant still used?

Reply:
Nope, I've removed it in the last commit.